test/src/page.spec.ts

```diff
@@ -1364,7 +1364,9 @@ describe('Page', function () {
       const {page, server} = await getTestState();
 
       const [error] = await Promise.all([
-        waitEvent<Error>(page, 'pageerror'),
+        waitEvent<Error>(page, 'pageerror', _error => {
+          return _error.message.includes('Fancy');
```

Comment:
What other error are you seeing here?
Prefixing with `_` shouldn't be done as ESLint will ignore some rules for variable starting with it.
```suggestion
        waitEvent<Error>(page, 'pageerror', err => {
          return err.message.includes('Fancy');
```

Reply:
I wasn't able to track them down (no stack trace), but the message was `Permission denied to access property "length"`. I only have those on CI